iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/string_internal.hpp

```diff
@@ -99,6 +104,9 @@ struct GetSize<std::string>
     }
 };
 
+// AXIVION Next Construct AutosarC++19_03-A0.1.6
+// AXIVION Next Construct AutosarC++19_03-A0.1.4 : it is a helper struct to ease the template
+// meta programming.
```

Comment:
@mossmaurice @saif-at-github I think we can also disable this rule globally since we already enabled the compiler warning for unused parameters and use `IOX_MAYBE_UNUSED` for all the cases we need the parameter due to SFINEA, etc.

The rule also explicitly excludes unnamed parameter but we actually want to keep the name and tagged them with `IOX_MAYBE_UNUSED`.

Reply:
Agree. disabled globally